.github/workflows/fossa.yaml

```diff
@@ -4,7 +4,8 @@ on:
   push:
     branches:
       - main
-
+  schedule:
+    - cron: "0 */12 * * *"
```

Comment:
So FOSSA action will run BOTH on merge to master AND according to the schedule?

Reply:
@andrejatcastai this schedule ensures that every 12 hours the FOSSA action will run on the main branch, so we will proactively get new things on FOSSA instead of waiting code changes.